src/app.js

```diff
@@ -0,0 +1,20 @@
+const elasticsearch = require('elasticsearch');
+
+async function indexData(action, settings) {
+    const connectionString = settings.connectionString
+    const client = new elasticsearch.Client({node: `${connectionString}`})
+    const res = await client.index(action.params.obj)
+    return res
+}
+
+async function searchData(action, settings) {
+    const connectionString = settings.connectionString
+    const client = new elasticsearch.Client({node: `${connectionString}`})
```

Comment:
These 2 lines are returning on every function. Please move them to an external function that will return the client object